lib/percy/cli/snapshot.rb

```diff
@@ -32,6 +32,8 @@ def run_snapshot(root_dir, options = {})
 
         base_resource_options = {strip_prefix: strip_prefix, baseurl: baseurl}
 
+        Percy::Cli::Client.new
```

Comment:
Having a free-floating `Percy::Cli::Client.new` here looks like an unused variable, if I saw this line in the future I would probably just remove it thinking it wasn't used — it's unclear that this has side-effects we want. 

A potential alternative (that's maybe too simple) is to just do this here:
```
Percy.client(client_info: "percy-cli/#{Percy::Cli::VERSION}", environment_info: '')
```

Though this may not be great because it makes testing unclear and can't use the tests you have now. Perhaps you can add a more explicit class-level method like `Percy::Cli::Client.setup_user_agent` that does it and making it a class method makes it clear that the user-agent state is not stored on the object.